sos/report/plugins/foreman_proxy.py

```diff
@@ -38,6 +38,14 @@ def setup(self):
                 self.apachepkg),
             "/var/log/{}*/katello-reverse-proxy_error_ssl.log*".format(
                 self.apachepkg),
+            "/var/log/{}*/rhsm-pulpcore-https-443_access_ssl.log*".format(
+                self.apachepkg),
+            "/var/log/{}*/rhsm-pulpcore-https-443_error_ssl.log*".format(
+                self.apachepkg),
+            "/var/log/{}*/rhsm-pulpcore-https-8443_access_ssl.log*".format(
+                self.apachepkg),
+            "/var/log/{}*/rhsm-pulpcore-https-8443_error_ssl.log*".format(
+                self.apachepkg),
         ])
```

Comment:
ACK to adding this. Maybe we can shrink it to `rhsm-pulpcore-https-*_access_ssl.log*` and `rhsm-pulpcore-https-*_error_ssl.log*` ? Just in case of port change?

Reply:
I wonder why be this specific? Foreman always controls httpd when present so it should be safe to grab everything under `/var/log/{}*/*` and reduce the churn you need whenever we make changes. I'd also want to see what @ekohl thinks

Reply:
Honestly, I'd expect such a thing in the apache plugin. That this is in the foreman_proxy pains me a lot since it has nothing to do with a Foreman Proxy. It just happens that on some systems there is also an Apache deployed which has some config that relates to it.

Reply:
We've explicitly moved these kinds of collections out of `apache` because of historical issues with multiple different components logging there, and the `apache` plugin either becoming unruly in its maintenance or insufficient in its handling of component-specific logs.

I do agree however, let's replace `8443` with a `*` glob so we don't miss these on non-standard configurations.